cp3pt0-deployment/common/utils.sh

```diff
@@ -862,4 +862,18 @@ function scale_up() {
         msg "Scaling up operand-deployment-lifecycle-manager deployment in ${operator_ns} namespace back to 1"
         scale_deployment ${operator_ns} operand-deployment-lifecycle-manager 1
     fi
+}
+
+function accept_license() {
+    local kind=$1
+    local namespace=$2
```

Comment:
```suggestion
    local namespace=$2
    local cr_name=$3
```

Suggest adding new cr_name variable since we know all the names of the CRs when running the scripts. Then the function can just patch the CR unconditionally, no need to run a get anymore.

Cert-manager creates CR named `default`
cs-operator creates CR named `common-service`
licensing-operator creates CR named `instance`
script creates NSS CR named `common-service`